.github/workflows/build-latest-docker-image.yaml

```diff
@@ -32,20 +37,28 @@ jobs:
         run: |
           git fetch --unshallow
 
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-region: us-east-2
+          role-to-assume: arn:aws:iam::499786161782:role/ghwf-axelar-core
```

Comment:
we should have this value in a secret env variable. Someone who have access to set secrets should do that before merging this in

Reply:
It's a bit late for that now, this value is now already public. Can we change it once it's a secret?

Reply:
we can but it would break naming convention. can we force push here and get rid of the name ?